data/cacheRepository.go

```diff
@@ -32,22 +33,30 @@ func (r *CacheRepository) SetCache(key string, value []byte) (err error) {
 	defer c.Close()
 	c.Send("MULTI")
 	c.Send("SET", key, value)
-	c.Send("EXPIRE", key, r.TTL)
+	c.Send("SET", key+":fresh", value)
```

Comment:
This change should be backwards compatible. The other places in the REST API using SetCache should work without setting the extra `:fresh` versions. Probably a new set of functions would be easiest (e.g., SetCachePair / SetFreshCache)